packages/salesforcedx-sobjects-faux-generator/src/generator/configUtil.ts

```diff
@@ -9,23 +9,33 @@ import {
   ConfigAggregator,
   ConfigFile,
   ConfigValue,
-  StateAggregator
+  OrgConfigProperties
 } from '@salesforce/core';
 import * as path from 'path';
 
-const defaultUserNameKey = 'defaultusername';
+async function getConfigAggregator(
```

Comment:
Nit: can we add a TODO with maybe the consolidation ticket number to ensure we find all these when we do the consolidation ticket?